invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/components/MostLoanedDocuments/state/actions.js

```diff
@@ -0,0 +1,33 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { document as documentApi } from '../../../../../../common/api';
+import { sendErrorNotification } from '../../../../../../common/components/Notifications';
+
+export const fetchMostLoanedDocuments = () => {
+  return async dispatch => {
+    dispatch({
+      type: IS_LOADING,
+    });
+
+    await documentApi
+      .list(
+        documentApi
+          .query()
+          .currentlyOnLoan()
+          .withSort('mostloaned')
+          .qs()
+      )
+      .then(response => {
```

Comment:
Here we decided to change Promises with only `await`/`async`, can you please change?